cosmo_tester/test_suites/image_based_tests/component_test.py

```diff
@@ -0,0 +1,170 @@
+########
+# Copyright (c) 2020 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import retrying
+
+from cosmo_tester.framework import util
+from cosmo_tester.framework.examples import get_example_deployment
+
+
+def _infra(image_based_manager, ssh_key, logger, tenant, test_config):
```

Comment:
How about moving this test (and the capabilities one) out into cosmo_tester/test_suites/component (or /capabilities), then you can define your three examples as fixtures in a conftest.py and not have to put as much code into the tests themselves.

Reply:
(even without that, they should probably be in their own folder to make for a more clear separation of tests)

Reply:
Ah, I see, looks like you can't put them into fixtures as you're passing the tenant in. Still, moving them off into their own folder would be good.